test/NuGet.Core.Tests/NuGet.Frameworks.Test/CompatibilityListProviderTests.cs

```diff
@@ -232,5 +232,48 @@ public void CompatibilityListProvider_NetStandard20Supporting()
             // count
             Assert.Equal(19, actual.Length);
         }
+        [Fact]
+        public void CompatibilityListProvider_NetStandard21Supporting()
+        {
+            // Arrange
+            var provider = CompatibilityListProvider.Default;
+
+            // Act
+            var actual = provider
+                .GetFrameworksSupporting(FrameworkConstants.CommonFrameworks.NetStandard21)
+                .Select(f => f.ToString())
+                .ToArray();
+
+            // Assert
+            // positive
+            Assert.Contains(".NETCoreApp,Version=v3.0", actual);
+            Assert.Contains(".NETStandard,Version=v2.1", actual);
+            Assert.Contains(".NETStandardApp,Version=v2.1", actual);
+
+            // negative
+            Assert.DoesNotContain(".NETFramework,Version=v4.7", actual); // only the minimum support version is returned
```

Comment:
.NET Framewowrk does not have a relation with netstandard 2.1.

You don't need to test both 4.6 and v4.7 here, only test the highest one available. 

Also the comments:

> // only the minimum support version is returned
// versions that are too small are not returned 

are not relevant. 
There is no mapping between netframework and netstandard2.1 yet. 
So you only need 1 comment :

> // frameworks with no relationship are not returned

for all the negatives

Reply:
Thanks! I'll change for those.